Tests/MapboxMapsTests/Foundation/MapViewTests.swift

```diff
@@ -410,3 +301,97 @@ final class MapViewTests: XCTestCase {
+    var attributionURLOpener: MockAttributionURLOpener!
+    var mapView: MapView!
+    var window: UIWindow!
+    var metalView: MockMetalView!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        notificationCenter = MockNotificationCenter()
+        bundle = MockBundle()
+        bundle.infoDictionaryStub.defaultReturnValue = ["UIApplicationSceneManifest": []]
+        cameraAnimatorsRunnerEnablable = MockMutableEnablable()
+        displayLink = MockDisplayLink()
+        locationProducer = MockLocationProducer()
+        dependencyProvider = MockMapViewDependencyProvider()
+        dependencyProvider.notificationCenter = notificationCenter
+        dependencyProvider.bundle = bundle
+        dependencyProvider.cameraAnimatorsRunnerEnablable = cameraAnimatorsRunnerEnablable
+        dependencyProvider.makeDisplayLinkStub.defaultReturnValue = displayLink
+        dependencyProvider.makeLocationProducerStub.defaultReturnValue = locationProducer
+        orientationProvider = MockInterfaceOrientationProvider()
+        attributionURLOpener = MockAttributionURLOpener()
+        mapView = MapView(
+            frame: CGRect(origin: .zero, size: CGSize(width: 100, height: 100)),
+            mapInitOptions: MapInitOptions(),
+            dependencyProvider: dependencyProvider,
+            orientationProvider: orientationProvider,
+            urlOpener: attributionURLOpener)
+        window = UIWindow()
+        window.addSubview(mapView)
+
+        metalView = try XCTUnwrap(dependencyProvider.makeMetalViewStub.invocations.first?.returnValue)
+        // reset is required here to ignore the setNeedsDisplay() invocation during initialization
+        metalView.setNeedsDisplayStub.reset()
+    }
+
+    override func tearDown() {
+        metalView = nil
+        window = nil
+        mapView = nil
+        attributionURLOpener = nil
+        orientationProvider = nil
+        dependencyProvider = nil
+        locationProducer = nil
+        displayLink = nil
+        cameraAnimatorsRunnerEnablable = nil
+        bundle = nil
+        notificationCenter = nil
+        super.tearDown()
+    }
+
+    @available(iOS 13, *)
+    func testDisplayLinkResumedWhenSceneMovingToForeground() throws {
+        bundle.infoDictionaryStub.defaultReturnValue = ["UIApplicationSceneManifest": []]
+        mapView = MapView(
+            frame: CGRect(origin: .zero, size: CGSize(width: 100, height: 100)),
+            mapInitOptions: MapInitOptions(),
+            dependencyProvider: dependencyProvider,
+            orientationProvider: orientationProvider,
+            urlOpener: attributionURLOpener)
+
+        mapView.didMoveToWindow()
+
+        notificationCenter.post(name: UIScene.willEnterForegroundNotification, object: window.parentScene)
+
+        XCTAssertEqual(displayLink.$isPaused.setStub.invocations.map(\.parameters), [false])
+    }
+
+    @available(iOS 13, *)
```

Comment:
This doesn't work to exclude a unit test. You need a runtime check as shown in some of the deleted tests.